src/App.tsx

```diff
@@ -1,27 +1,117 @@
 import React from 'react';
 import './App.scss';
+import { Clock } from './components/Clock';
 
-const App: React.FC = () => {
-  const timerId: NodeJS.Timer = setInterval(() => {
-    const date: Date = new Date();
-
-    // eslint-disable-next-line
-    console.log(date.toLocaleTimeString());
-  }, 1000);
-
-  // eslint-disable-next-line
-  console.log(timerId);
-
-  return (
-    <div className="App">
-      <h1>React clock</h1>
-      <p>
-        Current time:
-        {' '}
-        {/* Print the time here instead of DevTools */}
-      </p>
-    </div>
-  );
-};
+const watchNames = [
+  'Omega',
+  'Breguet',
+  'Seiko',
+  'Rolex',
+  'Patek Philippe',
+  'Chopard',
+  'Cartier',
+  'Tissot',
+  'Hublot',
+  'Swiss',
+  'Orient',
+];
+
+class App extends React.Component {
+  state = {
+    isClockVisible: true,
+    clockName: watchNames[Math.round(Math.random() * 10)],
+  };
+
+  createRandomName() {
+    this.setState({
+      clockName: watchNames[Math.round(Math.random() * 10)],
+    });
+
+    const clockNameElem = document.querySelector<HTMLElement>('.Clock__name');
+
+    if (clockNameElem) {
+      clockNameElem.classList.add('Clock__name__slider');
+      setTimeout(() => {
+        clockNameElem.classList.remove('Clock__name__slider');
+      }, 500);
+    }
+  }
+
+  changeClockVisible() {
+    const clockFace = document.querySelector<HTMLElement>('.Clock__face');
```

Comment:
don't use querySelector in components, cause we don't work directly with DOM in React
you can add classes with needed styles by some condition
also, you can use classnames library for this

https://www.npmjs.com/package/classnames